### PR TITLE
Upate RBAC role descriptions for stage

### PR DESCRIPTION
**Pull request by spank50**
RHCLOUD-35164

### CODE REVIEW

File: configs/stage/roles/automation-analytics.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
 {
   "roles": [
     {
-      "name": "Automation Analytics Administrator",
-      "description": "An Automation Analytics Administrator role that grants ALL permissions.",
+      "name": "Automation Analytics administrator",
```
Comment (spank50) [inline configs/stage/roles/automation-analytics.json:4]: "name": "Automation Analytics Administrator"
"display_name": "Automation Analytics administrator"

File: configs/stage/roles/automation-analytics.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
       "access": [
         {
           "permission": "automation-analytics:*:*"
         }
       ]
     },
     {
-      "name": "Automation Analytics Editor",
-      "description": "An Automation Analytics Editor role that grants read-write permissions.",
+      "name": "Automation Analytics editor",
```
Comment (spank50) [inline configs/stage/roles/automation-analytics.json:16]: "name": "Automation Analytics Editor",
"display_name": "Automation Analytics editor",

File: configs/stage/roles/automation-analytics.json [hunk 3]
```diff
@@ -28,11 +28,11 @@
       ]
     },
     {
-      "name": "Automation Analytics Viewer",
-      "description": "An Automation Analytics Viewer role that grants read permissions.",
+      "name": "Automation Analytics viewer",
```
Comment (spank50) [inline configs/stage/roles/automation-analytics.json:31]: "name": "Automation Analytics Viewer",
"display_name": "Automation Analytics viewer",

File: configs/stage/roles/cost-management.json [hunk 4]
```diff
@@ -1,38 +1,38 @@
 {
     "roles": [{
-        "name": "Cost Administrator",
-        "description": "A cost management administrator role that grants read and write permissions.",
+        "name": "Cost administrator",
```
Comment (spank50) [inline configs/stage/roles/cost-management.json:3]: "name": "Cost Administrator",
"display_name": "Cost administrator",

File: configs/stage/roles/cost-management.json [hunk 5]
```diff
@@ -1,38 +1,38 @@
...
         "admin_default": true,
-        "version": 3,
+        "version": 4,
         "access": [{
             "permission": "cost-management:*:*"
         }]
     }, {
-        "name": "Cost Price List Administrator",
-        "description": "A cost management role that grants read and write permissions on cost models.",
+        "name": "Cost price list administrator",
```
Comment (spank50) [inline configs/stage/roles/cost-management.json:12]: "name": "Cost Price List Administrator",
"display_name": "Cost Price List administrator",

File: configs/stage/roles/cost-management.json [hunk 6]
```diff
@@ -1,38 +1,38 @@
...
+        "version": 6,
         "access": [{
             "permission": "cost-management:cost_model:*"
         }, {
             "permission": "cost-management:settings:*"
         }]
     }, {
-        "name": "Cost Price List Viewer",
-        "description": "A cost management role that grants read permissions on cost models.",
+        "name": "Cost price list viewer",
```
Comment (spank50) [inline configs/stage/roles/cost-management.json:22]: "name": "Cost Price List Viewer",
"display_name": "Cost Price List viewer",

File: configs/stage/roles/cost-management.json [hunk 7]
```diff
@@ -1,38 +1,38 @@
...
+        "version": 5,
         "access": [{
             "permission": "cost-management:cost_model:read"
         }, {
             "permission": "cost-management:settings:read"
         }]
     }, {
-        "name": "Cost Cloud Viewer",
-        "description": "A cost management role that grants read permissions on cost reports related to cloud sources.",
+        "name": "Cost cloud viewer",
```
Comment (spank50) [inline configs/stage/roles/cost-management.json:32]: "name": "Cost cloud Viewer",
"display_name": "Cost cloud viewer",

File: configs/stage/roles/inventory.json [hunk 8]
```diff
@@ -30,26 +30,26 @@
       ]
     },
     {
-      "name": "Inventory Hosts Viewer",
-      "description": "Be able to read Inventory Hosts data.",
+      "name": "Inventory Hosts viewer",
```
Comment (spank50) [inline configs/stage/roles/inventory.json:33]: "name": "Inventory Hosts Viewer",
"display_name": "Inventory Hosts viewer",

File: configs/stage/roles/inventory.json [hunk 9]
```diff
@@ -30,26 +30,26 @@
...
         {
           "permission": "inventory:hosts:read"
         }
       ]
     },
     {
-      "name": "Inventory Groups Administrator",
-      "display_name": "Workspace Administrator",
-      "description": "Be able to read and edit Workspace data.",
+      "name": "Inventory Groups administrator",
```
Comment (spank50) [inline configs/stage/roles/inventory.json:46]: "name": "Inventory Groups Administrator",

File: configs/stage/roles/provisioning.json [hunk 10]
```diff
@@ -1,16 +1,16 @@
 {
     "roles": [{
-        "name": "Launch Administrator",
-        "description": "A launch administrator role that grants read and write permissions.",
+        "name": "Launch administrator",
```
Comment (spank50) [inline configs/stage/roles/provisioning.json:3]: "name": "Launch Administrator",
"display_name": "Launch administrator",

**Review by spank50 — COMMENTED** [review]
"name":  entries updated to
"display_name": where appropriate

**Review by spank50 — COMMENTED** [review]
"display_name" updates made

File: configs/stage/roles/sources.json
Comment (rabbott01) [inline configs/stage/roles/sources.json:1]: @spank50 FYI, this one is missing the "s" in "source"

File: configs/stage/roles/insights.json
Comment (rabbott01) [inline configs/stage/roles/insights.json:1]: @spank50 - should references to advisor be capitalized? It looks like other roles for OCM have "Advisor" and other roles capitalize service names like "Remediations", "Inventory" etc.

File: configs/stage/roles/sources.json
Comment (spank50) [reply configs/stage/roles/sources.json:1]: Change made. Thanks.

File: configs/stage/roles/insights.json
Comment (spank50) [reply configs/stage/roles/insights.json:1]: Thanks Ryan. I only saw RHEL advisor which I changed to RHEL Advisor.

File: configs/stage/roles/content-sources.json [hunk 11]
```diff
@@ -2,7 +2,7 @@
     "roles": [
       {
         "name": "Repositories administrator",
-        "description": "Perform any available operation against any repositories resource.",
+        "description": "Perform any available operation on any repositories resource.",
```
Comment (petracihalova) [inline configs/stage/roles/content-sources.json:5]: please for this change is needed to increase the version

File: configs/stage/roles/provisioning.json [hunk 12]
```diff
@@ -1,16 +1,18 @@
...
-        "version": 2,
+        "version": 3,
         "access": [{
             "permission": "provisioning:*:*"
         }]
     }, {
         "name": "Launch Viewer",
-        "description": "An launch role that grants read permissions on launch reservation and related resources.",
+        "display_name": "Launch viewer",
+        "description": "Perform read operations on launch reservation and related resources.",
```
Comment (petracihalova) [inline configs/stage/roles/provisioning.json:15]: please for this change is needed to increase the version

**Review by petracihalova — CHANGES_REQUESTED** [review]

File: configs/stage/roles/content-sources.json (hunk 11 above)
Comment (spank50) [reply configs/stage/roles/content-sources.json:5]: "version": 2,   is now "version": 3,

File: configs/stage/roles/provisioning.json (hunk 12 above)
Comment (spank50) [reply configs/stage/roles/provisioning.json:15]: "version": 2,   
change to
"version": 3

File: configs/stage/roles/provisioning.json [hunk 13]
```diff
@@ -29,9 +31,10 @@
         }]
     }, {
         "name": "Launch on AWS User",
-        "description": "An AWS launch role that grants write permissions on launch reservation and related resources.",
+        "display_name": "Launch on AWS user",
+        "description": "Perform upate operations on AWS launch reservation and related resources.",
```
Comment (petracihalova) [inline configs/stage/roles/provisioning.json:35]:
```suggestion
        "description": "Perform update operations on AWS launch reservation and related resources.",
```

File: configs/stage/roles/provisioning.json [hunk 14]
```diff
@@ -43,9 +46,10 @@
         }]
     }, {
         "name": "Launch on Azure User",
-        "description": "An Azure launch role that grants write permissions on launch reservation and related resources.",
+        "display_name": "Launch on Azure user",
+        "description": "Perform upate operations on Azure launch reservation and related resources.",
```
Comment (petracihalova) [inline configs/stage/roles/provisioning.json:50]:
```suggestion
        "description": "Perform update operations on Azure launch reservation and related resources.",
```

File: configs/stage/roles/remediations.json [hunk 15]
```diff
@@ -2,10 +2,10 @@
   "roles": [
     {
       "name": "Remediations administrator",
-      "description": "Perform any available operation against any Remediations resource",
+      "description": "Perform any available operation on any Remediations resource",
```
Comment (petracihalova) [inline configs/stage/roles/remediations.json:5]:
```suggestion
      "description": "Perform any available operation on any Remediations resource.",
```

File: configs/stage/roles/ocm.json [hunk 16]
```diff
@@ -2,83 +2,83 @@
...
         "id": "OrganizationAdmin",
         "tenant": "ocm"
       }
     },
     {
       "name": "OCM Cluster Autoscaler Editor",
-      "display_name": "OCM Cluster Autoscaler Editor",
-      "description": "Grants permission to edit cluster autoscaler",
+      "display_name": "OCM cluster autoscaler editor",
+      "description": "Perform update operations on the cluster autoscaler",
```
Comment (petracihalova) [inline configs/stage/roles/ocm.json:78]:
```suggestion
      "description": "Perform update operations on the cluster autoscaler.",
```

**Review by petracihalova — APPROVED** [review]

**Review by rabbott01 — APPROVED** [review]

**Pull request merged**